src/main/java/mate/academy/spring/model/dto/response/ShoppingCartResponseDto.java

```diff
@@ -0,0 +1,26 @@
+package mate.academy.spring.model.dto.response;
+
+import java.util.List;
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
you should not put this annotations here
this annotation (and annotation Service, Repository) is used to put this class to spring control, and autowire it later
this class will be created manually

remove it from other dtos